lib/tasks/javascript.rake

```diff
@@ -133,6 +133,18 @@ end
 task "javascript:update_constants" => :environment do
   task_name = "update_constants"
 
+  auto_groups =
+    Group::AUTO_GROUPS.inject({}) do |result, (group_name, group_id)|
+      result.merge(
+        group_name => {
+          id: group_id,
+          automatic: true,
```

Comment:
Small one but I wonder about including this attribute considering that it will always be true and the constant name reflects that all groups are automatic.

Reply:
It's just because these will be used in tests to simulate actual `groups` on the user model, and this is how they are represented, since they can be mixed in with non-automatic groups.